Charts/Classes/Data/Implementations/Standard/ChartDataEntry.swift

```diff
@@ -38,30 +38,32 @@ public class ChartDataEntry: NSObject
         self.xIndex = xIndex
     }
     
-    public init(value: Double, xIndex: Int, data: AnyObject?)
+    public init(value: Double, xIndex: Int, data: Any?)
```

Comment:
I am not sure if this is the correct way to use Any. I am confused what Swift is trying to do here about Any and AnyObject. I know `isEqual` method in beta 6 uses Any type, however simply use `self.data = data as AnyObject` seems lame to me..